backend/src/common/enums/modelName/index.ts

```diff
@@ -0,0 +1 @@
+export { ModelName } from 'healthcare-shared/common/enums';
```

Comment:
We don't need this enum for all application, so you should place this enum only in the backend application